packages/studioCMS/src/coreIntegration.ts

```diff
@@ -137,6 +126,17 @@ export default defineIntegration({
 					// Log Setup Complete
 					studioLogger(LoggerOpts.logInfo, CoreStrings.SetupComplete);
 				},
+				'astro:config:done': async (params) => {
+					// Add Virtual DTS File
+					studioLogger(
+						(await studioLoggerOptsResolver(params.logger, resolvedOptions.verbose)).logInfo,
+						CoreStrings.AddVirtualDTS
+					);
+					params.injectTypes({
+						filename: 'core.d.ts',
```

Comment:
Hmm wonder if this should be something more like `studiocms-core.d.ts`?

Reply:
Each of these get put in the `.astro/integrations/` folder with a naming scheme unique to each one where core is `.astro/integrations/_astrolicious_studiocms/core.d.ts` or for the admin dashboard... `.astro/integrations/_astrolicious_studiocms_admindashboard/dashboard.d.ts` thanks to the new injectType helper from astro core

Reply:
side note, the future planned refactor will also be changing a bit of how all this is working to make things a bit easier and simpler!

Reply:
Ah very nice! Yeah that's fine that's cool!